Userland/Libraries/LibJS/Runtime/DateConstructor.cpp

```diff
@@ -187,6 +187,7 @@ static double parse_date_string(VM& vm, ByteString const& date_string)
         "%Y-%m-%e"sv,                          // "2024-1-15"
         "%Y-%m-%e%t%T%tGMT%z"sv,               // "2024-07-05 00:00:00 GMT-0800"
         "%d%t%B%t%Y"sv,                        // "01 February 2013"
+        "%d%t%b%t%Y%t%R"sv,                    // "01 Jan 2000 08:00"
```

Comment:
While you're here, could you also add `%d%t%b%t%Y` and `%d%t%B%t%Y%t%R`? They seem to work in both Chrome and Firefox as well.

Reply:
Added 👍